sippy/sippy/pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: sippy
+  namespace: sippy
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 900M
+  # storageClassName: standard
```

Comment:
@spiffxp - what storage class we have in the community cluster?

Reply:
@wojtek-t we have 2 storage classes in the cluster : 

```shell
kubectl get sc
NAME                 PROVISIONER            AGE
ssd                  kubernetes.io/gce-pd   334d
standard (default)   kubernetes.io/gce-pd   551d
```

Reply:
Thanks!

@deads2k - standard should be good enough for us;
 Though I would visibly increase the size, say to 10GB or sth.

Reply:
@deads2k - friendly ping